base/broadcast.jl

```diff
@@ -96,7 +96,7 @@ dumpbitcache(Bc::Vector{UInt64}, bind::Int, C::Vector{Bool}) =
             # reverse-broadcast the indices
             @nexprs $nargs i->(I_i = newindex(I, imap_i))
             # extract array values
-            @nexprs $nargs i->(@inbounds val_i = _broadcast_getvals(A_i, I_i))
```

Comment:
Since `unsafe_getindex` if now basically defined as `(@inbounds b = getindex(A, I); b)` for an `AbstractArray` `A` and indices `I`, I thought I could change it. If you think is better to use `@inbounds` and `@propagate_inbounds` I'll make the change.